src/pages/en/reference/directives-reference.md

```diff
@@ -164,7 +164,7 @@ By default, Astro will process, optimize, and bundle any `<script>` and `<style>
 
 The `is:inline` directive means that `<style>` and `<script>` tags:
 
-- Will not be bundled into an external file.
+- Will not be bundled into an external file. This means that attributes like `defer` which control the loading of an external file will have no effect.
```

Comment:
What do you think about linking to some more extensive external documentation here?

```suggestion
- Will not be bundled into an external file. This means that [attributes like `defer`](https://javascript.info/script-async-defer) which control the loading of an external file will have no effect.
```

Link I’m proposing: <https://javascript.info/script-async-defer>

Reply:
I like this, it's better than the official MDN docs for explaining this point!